setup.sh

```diff
@@ -4,6 +4,7 @@ KW_LIB_DIR='src'
 include "${KW_LIB_DIR}/lib/kwio.sh"
 include "${KW_LIB_DIR}/lib/kwlib.sh"
 include "${KW_LIB_DIR}/lib/kw_db.sh"
+include "${KW_LIB_DIR}/lib/help.sh"
```

Comment:
The correct path is `${KW_LIB_DIR}/help.sh` instead of `${KW_LIB_DIR}/lib/help.sh`. This causes the file to not be included, which breaks `kw --version` for the installed case (see screenshot below).

![Screenshot from 2023-12-23 17-19-11](https://github.com/kworkflow/kworkflow/assets/95055784/426a5053-b877-437f-a777-095a443106bf)